keps/sig-cloud-provider/20190215-promoting-cloud-provider-labels.md

```diff
@@ -0,0 +1,166 @@
+---
+title: Promoting Cloud Provider Labels to GA
+authors:
+  - @andrewsykim
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - sig-node
+  - sig-storage
+reviewers:
+  - @dims
+  - @liggit
+  - @msau42
+  - @saad-ali
+  - @thockin
+approvers:
+  - @thockin
+  - @liggit
+editor: TBD
+creation-date: 2019-02-15
+last-updated: 2019-02-15
+status: provisional
+see-also:
+  - "/keps/sig-node/20190130-node-os-arch-labels.md"
+---
+
+# Promoting Cloud Provider Labels to GA
+
+## Table of Contents
+
+   * [Promoting Cloud Provider Labels to GA](#promoting-cloud-provider-labels-to-ga)
+      * [Table of Contents](#table-of-contents)
+      * [Release Signoff Checklist](#release-signoff-checklist)
+      * [Summary](#summary)
+      * [Motivation](#motivation)
+         * [Goals](#goals)
+         * [Non-Goals](#non-goals)
+      * [Proposal](#proposal)
+         * [Implementation Details/Notes/Constraints [optional]](#implementation-detailsnotesconstraints-optional)
+         * [Risks and Mitigations](#risks-and-mitigations)
+      * [Design Details](#design-details)
+         * [Test Plan](#test-plan)
+         * [Graduation Criteria](#graduation-criteria)
+            * [Examples](#examples)
+               * [Alpha -&gt; Beta Graduation](#alpha---beta-graduation)
+               * [Beta -&gt; GA Graduation](#beta---ga-graduation)
+         * [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+         * [Version Skew Strategy](#version-skew-strategy)
+      * [Implementation History](#implementation-history)
+      * [Drawbacks [optional]](#drawbacks-optional)
+      * [Alternatives [optional]](#alternatives-optional)
+
+## Release Signoff Checklist
+
+- [X] kubernetes/enhancements issue in release milestone, which links to KEP (this should be a link to the KEP location in kubernetes/enhancements, not the initial KEP PR)
+- [ ] KEP approvers have set the KEP status to `implementable`
+- [ ] Design details are appropriately documented
+- [ ] Test plan is in place, giving consideration to SIG Architecture and SIG Testing input
+- [ ] Graduation criteria is in place
+- [ ] "Implementation History" section is up-to-date for milestone
+- [ ] User-facing documentation has been created in [kubernetes/website], for publication to [kubernetes.io]
+- [ ] Supporting documentation e.g., additional design documents, links to mailing list discussions/SIG meetings, relevant PRs/issues, release notes
+
+## Summary
+
+When node and volume resources are created in Kubernetes, labels may be applied based on the underlying cloud provider of the Kubernetes cluster.
```

Comment:
nit. on the may, should, must scale (https://www.ietf.org/rfc/rfc2119.txt) it seems like this is a should.